### PR TITLE
Updating pagination component

### PR DESCRIPTION
**Pull request by srwang**
-updating pagination component to accept custom elements for chevron
-updating width so that 1140px is usable space, not total including padding

### CODE REVIEW

File: src/Pagination/index.jsx [hunk 1]
```diff
@@ -417,6 +417,8 @@ The default is:
 }
 ``` */
   buttonLabels: PropTypes.shape({
+    leftChevron: PropTypes.element,
```
Comment (abutterworth) [inline src/Pagination/index.jsx:420]:
```suggestion
    leftChevron: PropTypes.node,
```
(any renderable element)

File: scss/core/_variables.scss [hunk 2]
```diff
@@ -121,7 +121,7 @@ $container-max-widths: (
   sm: 540px,
   md: 720px,
   lg: 960px,
-  xl: 1140px
+  xl: 1140px + 2 * $grid-gutter-width,
```
Comment (abutterworth) [inline scss/core/_variables.scss:124]: This hasn't been defined yet, best to keep this change out of `core/*` and bring it into `edx/*`

File: scss/core/_variables.scss [hunk 3]
```diff
@@ -121,7 +121,7 @@ $container-max-widths: (
   sm: 540px,
   md: 720px,
   lg: 960px,
-  xl: 1140px
+  xl: 1140px,
```
Comment (abutterworth) [inline scss/core/_variables.scss:124]:
```suggestion
  xl: 1140px
```
left over addition

File: scss/edx/overrides/_grid.scss [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
 .container-fluid {
-  max-width: 1140px;
+  max-width: 1140px + 2 * $grid-gutter-width;
   padding-left: $grid-gutter-width;
   padding-right: $grid-gutter-width;
 }
+
+$container-max-widths: map-merge(
+  (
+    xl: 1140px + 2 * $grid-gutter-width,
```
Comment (abutterworth) [inline scss/edx/overrides/_grid.scss:14]: Does this also require us to change: `$grid-breakpoints`? https://github.com/edx/paragon/pull/534/files#diff-252b35fc7cc2964fadc678c056766291R109?

Comment (srwang) [reply scss/edx/overrides/_grid.scss:14]: grid-breakpoints seems to be set to the correct width already, so I leaned on the side of writing less code

Comment (abutterworth) [reply scss/edx/overrides/_grid.scss:14]: 👍

Comment (abutterworth) [reply scss/edx/overrides/_grid.scss:14]: 👍

**Pull request merged**